src/normal.cc

```diff
@@ -226,8 +226,20 @@ void goto_commands(Context& context, NormalParams params)
                 select<mode, select_to_line_begin<true>>(context, {});
                 break;
             case 'i':
-                select<mode, select_to_first_non_blank>(context, {});
```

Comment:
select_to_first_non_blank is only used here, so it would make more sense to modify that function.

Reply:
But my modifications don't select the first whitespace character, so I should probably rename that function as well?